hack/debug.sh

```diff
@@ -34,6 +37,29 @@ filter_env () {
     awk '/"env": \[$/ { indent = index($0, "\""); skipping = 1; next } !skipping { print; } skipping { if (index($0, "]") == indent) skipping = 0; }'
 }
 
+log_system () {
+    logpath=$1
+
+    echo_and_eval  journalctl --boot                                  &> $logpath/journal-full
+    echo_and_eval  journalctl -u $aos_master_service                  &> $logpath/journal-openshift
+    echo_and_eval  systemctl show $aos_master_service                 &> $logpath/systemctl-show
```

Comment:
Note that this needs to be `$aos_master_service` for the master, but `$aod_node_service` for the node